### DISCUSSION TITLE
And by analogy with nsfw, add the ability to disable cruelty, blood and other content unpleasant to many people

### DISCUSSION THREAD

**Discussion by Ultron55** [civitai/civitai#514] [Ideas]
And by analogy with nsfw, add the ability to disable cruelty, blood and other content unpleasant to many people.
Exampe https://civitai.com/models/15003/cyberrealistic
Under this model, in galea, there are posts of normal users, and then a whole bunch of bloody perverted monsters from the AvaCadava user https://civitai.com/user/AvaCadava.
I don't want to see this.
There is an option to disable violence in the settings, but there are no mutilated bloody bodies.

**Comment by Judas87Swiff** [comment]
I'm pretty sure this is implemented already, when signing up recently it certainly mentioned asking what my preferences for different levels of content, including violence, blood, gore, and a bunch of other nasties.

**Reply by Ultron55** [reply]
almost a year has passed since then
